src/resources/postcss/tickets.pcss

```diff
@@ -27,6 +27,60 @@
 	}
 }
 
+/*
+	Tribe Horizontal Datepicker (Day, Month, Year)
+*/
+.tribe_horizontal_datepicker__container {
+	label {
+		width: 100%;
+		display: inline-block;
+	}
+	.tribe_horizontal_datepicker {
+		width: 100%;
+		margin: var(--spacer-1) 0;
+		display: inline-block;
+
+		@media(--viewport-medium) {
+			width: calc( ( 100% / 3 ) - var(--spacer-1) - 4px ); /* 4px = safety margin */
+			margin-right: var(--spacer-1);
+			display: inline-block;
+		}
+
+		select {
+			border-color: var(--form-color-border-secondary);
```

Comment:
You can set border-color, border-style, and border-width all in one border style.

Reply:
fixed